packages/apps-v5/src/commands/apps/destroy.js

```diff
@@ -22,7 +22,7 @@ async function run(context, heroku) {
   if (git.inGitRepo()) {
     // delete git remotes pointing to this app
     _(await git.listRemotes())
-      .filter((r) => git.gitUrl(app) === r[1] || git.sshGitUrl(app) === r[1])
+      .filter((r) => git.gitUrl(app) === r[1])
```

Comment:
Do you think it would be worth leaving the ssh url here so any old remotes that do use that format get removed?

Reply:
Yeah, that does makes sense. Will update, thx!

Reply:
Yeah, that does makes sense. Will update, thx!

Reply:
@saracope thanks! You'll also need to bring back that `git.sshGitUrl` function. Are there tests for this command? Weird that that didn't cause a failure.